src/silx/gui/plot/ImageStack.py

```diff
@@ -456,6 +511,33 @@ def createUrlIndexes():
                 first_url = self._urls[list(self._urls.keys())[0]]
                 self.setCurrentUrl(first_url)
 
+    def _urlsRemoved(self, urls: tuple) -> None:
+        """
+        remove provided urls from the given one and reset urls
+
+        :param tuple urls: urls as str
+        """
```

Comment:
No need to give twice the type in the signature and the docstring

```suggestion
        """
        Remove provided URLs from the given one and reset URLs

        :param urls: URLs as str
        """
```